includes/admin/payments/actions.php

```diff
@@ -67,6 +67,7 @@ function give_update_payment_details( $data ) {
 
 	$curr_total = $payment->total;
 	$new_total  = give_maybe_sanitize_amount( $data['give-payment-total'] );
+	$new_total  = is_numeric( $new_total ) ? $new_total : 0;
```

Comment:
@jaydeeprami Instead of using this condition  use this `$new_total  = absint(give_maybe_sanitize_amount( $data['give-payment-total'] ))`

Reply:
@ravinderk  I have applied changes.